.happy/terraform/envs/rdev/main.tf

```diff
@@ -8,24 +8,28 @@ module "stack" {
   deployment_stage    = "rdev"
   stack_prefix        = "/${var.stack_name}"
   k8s_namespace       = "jheath-rdev-happy-happy-env"
+  routing_method      = "CONTEXT"
```

Comment:
this is the setting that lets us use both the services below under the same domain